scitt/verify_receipt_signature.py

```diff
@@ -97,8 +98,12 @@ def verify_receipt(receipt: bytes) -> bool:
     """
 
     # decode the cbor encoded cose sign1 message
-    message = Sign1Message.decode(receipt)
-
+    try:
+        message = Sign1Message.decode(receipt)
+    except: 
```

Comment:
This will catch keyboard interrupt and you will have to kill 9 the task if things go wrong.
Idiomatic way to do this is to 

     except Exception as ex:
         print("failed to decode receiot %s", ex)

and this does not catch ctrl-c and suchlike.

Reply:
added more specific exceptions to catch